File: Semester3/MAP/Seminare/Sem11/Sem11/domain/Entitys/Sarcina.cs
namespace Sem11;

public class Sarcina : Entity<String>
{
    private Dificultate dificultate;
    private int nrOreEstimate;

    public Sarcina(Dificultate dificultate, int nrOreEstimate)
    {
        this.dificultate = dificultate;
        this.nrOreEstimate = nrOreEstimate;
    }

    public Dificultate Dificultate { get => dificultate; set => dificultate = value; }
    
    public int NrOreEstimate { get => nrOreEstimate; set => nrOreEstimate = value; }

    public override string ToString()
    {
        return $"{Dificultate} - {NrOreEstimate}";
    }
}

File: Semester3/MAP/Seminare/Seminar14/src/main/java/org/example/seminar14/Domain/Sarcina.java
package org.example.seminar14.Domain;

public class Sarcina {
    private int id;
    private Dificultate dificultate;
    private int nrOreEstimate;

    public Sarcina(Dificultate dificultate, int nrOreEstimate) {
        this.dificultate = dificultate;
        this.nrOreEstimate = nrOreEstimate;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public Dificultate getDificultate() {
        return dificultate;
    }

    public void setDificultate(Dificultate dificultate) {
        this.dificultate = dificultate;
    }

    public int getNrOreEstimate() {
        return nrOreEstimate;
    }

    public void setNrOreEstimate(int nrOreEstimate) {
        this.nrOreEstimate = nrOreEstimate;
    }
}
